Draft of the blog post in Spanish (#1022)
ropensci/roweb3: content/blog/2025-05-27-champions-histories/index.en.md

---

> title: Carolina Pradier, champion in learning to teach.

Author: I like it, thanks!

author:

> - Yanina Bellini Saibene

Reviewer: In the Stars interviews, the interviewees are also authors. Is that something you'd like to do here too?

Author: In the Star, not all of them are authors. We asked some of them to suggest changes to the text and translate it into their native language.  I don't remember if, after the fact, we added all of them as authors. 
However, I do like the Champions as authors if they agree. They review everything before we publish.

- Alejandra Bellini
date: '2025-05-27'
slug: champions-histories-carolina-pradier
editor:
- Steffi LaZerte
tags:
- packages
- champions program
- community

> - interview

Reviewer: The Stars series uses plural, and if this is a series, let's give it it's own tag, like with the stars!
```suggestion
- interviews
- champions-stories
```

description: Stories of Champions Series. Our first story is that of Carolina Pradier, main maintainer of the eph package. We review her participation in the program and the impact of her work in different organizations and communities.
params:
  doi: 10.59350/f7jvs-8js04 
---

> 

Reviewer: Do you want to add an introductory sentence explaining that this is part of a series of posts, like you state in the description (and at the start of the Stars posts)

```suggestion
This is our first story in our interview series **Stories of Champions**, where we introduce you to different Champions and the impact they have. Our first Champion is [Carolina Pradier](/es/author/carolina-pradier/), main maintainer of the eph package.

```

> [Carolina Pradier](/es/author/carolina-pradier/) has long worked to make the world a more equitable place. Her doctoral research focused on inequalities in science with a focus on the mechanisms that structure gender inequality in the Latin American scientific community. She is also part of [Ecofeminita](https://ecofeminita.com/) -Asociación Civil Economía Feminista-, an interdisciplinary organization created and led by women.

Reviewer: Assuming we use the full name in the introductory paragraph...

```suggestion
Carolina has long worked to make the world a more equitable place. Her doctoral research focused on inequalities in science with a focus on the mechanisms that structure gender inequality in the Latin American scientific community. She is also part of [Ecofeminita](https://ecofeminita.com/) -Asociación Civil Economía Feminista-, an interdisciplinary organization created and led by women.
```

It was in this sense that Carolina [organized an event in Spanish](https://vimeo.com/899372049) with news and ideas for the use of its epa package, achieving a double objective: sharing content about R packages in Spanish and bringing more people closer to the community.
It also participated in a [Community Call on the rOpenSci Champions Program.](/commcalls/july2023-championprogram/) where he shared his experience together with mentor [Beatriz Milz](/es/author/beatriz-milz/) and his cohort partner, [Victor Ordu](/author/victor-ordu/).

## Perfecting a job well done

The [eph package](https://github.com/ropensci/eph/) package allows processing data from the Permanent Household Survey - INDEC, Argentina to replicate or obtain conclusions independent of those published in the reports prepared by the government agency. It also allows the analysis of different categories and indicators not provided in the official reports.
Carolina explained that throughout the program they managed to improve the package:

> "We tried to improve the quality of the code (reliability and efficiency), we improved the documentation and created new functions.

> They also worked hard to translate the documentation and make it more accessible to all non-English speaking users.

Author: Take into account that this is the text of the automatic translation, so DeepL made mistakes.  :-) 
In Spanish didn't say "all" :-P (that is why sometimes Carolina is a she, a he, or a it in the text...assumption of the  model :-P )

> They also worked hard to translate the documentation and make it more accessible to all non-English speaking users.

Reviewer: It's not only the 'all', but the implication that improves accessibility generally to non-English speakers. Do you mean to Spanish speaking users? If it's something that makes it more accessible to multiple languages, that's exceptionally cool and should be elaborated on 😁

> They also worked hard to translate the documentation and make it more accessible to all non-English speaking users.

Author: It is Spanish speaking user :-)

> During the peer review process of rOpenSci, [demonstrated that the way of working can be rigorous and warm at the same time.](https://github.com/ropensci/software-review/issues/593#issuecomment-1709968472)

Author: The comment is more for the review process than the building of the package, what about this: 

The rOpenSci peer review process showed how a rigorous approach can also be welcoming and kind.

> During the peer review process of rOpenSci, [demonstrated that the way of working can be rigorous and warm at the same time.](https://github.com/ropensci/software-review/issues/593#issuecomment-1709968472)

Reviewer: That's perfect!

> During the peer review process of rOpenSci, [demonstrated that the way of working can be rigorous and warm at the same time.](https://github.com/ropensci/software-review/issues/593#issuecomment-1709968472)

Reviewer: I like your suggestion better than mine 😆

{{< figure src = "revision.png" alt = "Comentario en el issue de revisión de eph: Antes que nada, felicitaciones por el paquete. Es un paquete sumamente útil para la comunidad y uno de los más importantes para aquellos que trabajan en política pública. Además de que es un ejemplo para otros organismos del estado de cómo se puede disponibilizar data que sea en servicio de la comunidad. Además, es una linda muestra de cómo se puede mejorar un proyecto con el tiempo y no sólo se sostiene en el mismo sino que está permanentemente creciendo." caption = "Comentario abierto, transparente y amigable de la revisión por pares en el proceso de rOpenSci del paquete eph">}}

## Open science

> The package is a tool for academic research, journalism and evidence-based public policy formulation. The champions program allowed us to add quality, improve the software and add different points of view through peer and expert review.

Author: If that is the impression, then it is wrong. That wasn't the idea. Thank you for reviewing.

Once again it was able to meet another of the objectives set at the beginning of the rOpenSci program:

> I am particularly excited about the prospect of participating in a program aimed at making the R community more inclusive. in addition, I feel that this experience would be a great opportunity as it would allow me to meet and work with highly skilled professionals whose work I deeply admire. Not only am I interested in learning from their experience, but I also believe that building these networks can help increase the impact of my community's efforts and access new sources of support."

The package further inspired other programmers, data scientists, and people from different disciplines to develop in Python the [PyEPH library](https://pyeph.readthedocs.io/es/latest/) library, which has a similar purpose to the epa package in R, but to be used and supported by people in the Python data community.

> ## Beyond learning: building in community

Author: We want to say building something with other people (in Spanish we say "en comunidad" and the translation use "in" but perhaps is "within"). We didn't mean building a community.

> ## Beyond learning: building in community

Reviewer: I think 'with' then would be the best for the idea of building together with other people. "building with a community"

Carolina Pradier was able to learn about R, sort her knowledge, acquire new skills and offer a multilingual package so that more people can access open science. The contact with her mentor, with her peers, with the people who participated in the review process and the people who use the package was a great contribution to learning and personal growth. Her participation in the Champions Program was the beginning of a network of links that promise new projects.

> I always had someone to talk to to help me solve problems; it was never a frustrating process."

was Carolina's response to the question about possible improvements to be made to the Campeones y Campeonas program, and she added:

> I think the team is very well organized to help you. During the training I got a lot of information on how to solve the problems that I could encounter".

> Shared knowledge allows us to anticipate, help and include. The more experiences like Carolina Pradier's we can have the more shared open science there will be.

Author: Love it!

author:
- Yanina Bellini Saibene
- Alejandra Bellini
date: '2025-05-27'
slug: champions-histories-carolina-pradier
editor:
- Steffi LaZerte
tags:
- packages
- champions program
- community
- interviews
- champions-stories
description: Stories of Champions Series. Our first story is that of Carolina Pradier, main maintainer of the eph package. We review her participation in the program and the impact of her work in different organizations and communities.
params:
  doi: 10.59350/f7jvs-8js04 
---
This is our first story in our interview series **Stories of Champions**, where we introduce you to different Champions and the impact they have. Our first Champion is [Carolina Pradier](/es/author/carolina-pradier/), main maintainer of the eph package.

[Carolina Pradier](/author/carolina-pradier/) has long worked to make the world a more equitable place. Her doctoral research focused on inequalities in science with a focus on the mechanisms that structure gender inequality in the Latin American scientific community. She is also part of [Ecofeminita](https://ecofeminita.com/) (Asociación Civil Economía Feminista), an interdisciplinary organization created and led by women.

> Carolina has a degree in Economics from the University of Buenos Aires and a holds a Master's degree in Social Sciences of Work from the Faculty of Social Sciences of the University of Buenos Aires. She has a PhD in Information Science from the University of Montreal and teaches at post-secondary institutions.

Author: ```suggestion
Carolina has a degree in Economics from the University of Buenos Aires (UBA) and a Master's degree in Labour Studies from the Faculty of Social Sciences (UBA). She is a PhD candidate in Information Science at the University of Montreal and teaches at post-secondary institutions.
```

> Carolina has a degree in Economics from the University of Buenos Aires and a holds a Master's degree in Social Sciences of Work from the Faculty of Social Sciences of the University of Buenos Aires. She has a PhD in Information Science from the University of Montreal and teaches at post-secondary institutions.

Author: Yes, another translation issue, in the Spanish text said "Doctoranda".

> As a researcher, Carolina was part of the Center for Studies on Population, Employment and Development and now works in the [Open Science research team at UNESCO](https://www.unesco.org/en/open-science). She also is the maintainer of the [eph](https://docs.ropensci.org/eph/) R package, whose objective is to facilitate working with the Permanent Household Survey data.

Author: ```suggestion
As a researcher, Carolina was part of the Center for Studies on Population, Employment and Development and now works for the [UNESCO Chair on Open Science](https://www.unesco.org/en/open-science). She also is the maintainer of the [eph](https://docs.ropensci.org/eph/) R package, whose objective is to facilitate working with the Permanent Household Survey data.
```

> {{< figure src = "eph-docs.png" alt = "Documentacion oficial del paquete eph con el template de rOpenSci" caption = "Documentación oficial del paquete eph">}}

Author: ```suggestion
{{< figure src = "eph-docs.png" alt = "Official documentation of the eph package with the rOpenSci template">}}
```

It was with this goal that Carolina [organized an event in Spanish](https://vimeo.com/899372049) with news and ideas for the use of the eph package. This achieved a double objective: sharing content about R packages in Spanish and bringing more people closer to the R and open science community.
Carolina also participated in a [Community Call on the rOpenSci Champions Program.](/commcalls/july2023-championprogram/) where she shared her experiences alongside mentor [Beatriz Milz](/es/author/beatriz-milz/) and cohort partner, [Victor Ordu](/author/victor-ordu/).

## Perfecting a job well done

The [eph](https://github.com/ropensci/eph/) package processes data from the Permanent Household Survey ([INDEC](https://www.indec.gob.ar)) in Argentina to replicate or conduct independent analyses from those published in the reports prepared by the government agency. It also allows the analysis of different categories and indicators not provided in the official reports.
Carolina explained that throughout the program they managed to improve the package:

> "We tried to improve the quality of the code (reliability and efficiency), we improved the documentation and created new functions.

> They also worked hard to translate the documentation and make it more accessible to all non-English speaking users.

Author: ```suggestion
They also worked hard to translate the documentation and make it more accessible to all Spanish-speaking users.
```
